how-it-works/1_protocol/00_protocol_overview.card.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Overview
+---
+
+![](/img/how-it-works/core_protocol_layers.png)
+
+# Overview
+
+The Internet Computer is powered by the Internet Computer Protocol (ICP), from which its utility token, the ICP token, derives its name.
+The core part of the IC protocol, the *core IC protocol*, is a 4-layer protocol that is running on the nodes of each subnet.
+By running the core IC protocol, the nodes of a subnet realize a blockchain-based replicated state machine that makes progress independently of the other subnets, each subnet's nodes running an instance of the same IC core protocol.
```

Comment:
Can we just delete ", each subnet's node running..."? It seems like we already said that